src/main/java/frc/robot/components/TrigonTalonFX.java

```diff
@@ -39,21 +36,4 @@ public TrigonTalonFX(int id, MotorConfig motorConfig) {
     public TrigonTalonFX(int id) {
         this(id, new MotorConfig());
     }
-
-    public ErrorCode configAllSettings(double openLoopRamp, double closedLoopRamp, PIDCoefs pidCoefs, int timeoutMs) {
-        TalonFXConfiguration config = new TalonFXConfiguration();
-        config.openloopRamp = openLoopRamp;
-        config.closedloopRamp = closedLoopRamp;
-        config.slot0.kP = pidCoefs.getKP();
-        config.slot0.kI = pidCoefs.getKI();
-        config.slot0.kD = pidCoefs.getKD();
-        config.slot0.kF = pidCoefs.getKF();
-        config.slot0.allowableClosedloopError = (int) pidCoefs.getTolerance();
-
-        return this.configAllSettings(config);
-    }
-
-    public ErrorCode configAllSettings(double openLoopRamp, double closedLoopRamp, PIDCoefs pidCoefs) {
-        return configAllSettings(openLoopRamp, closedLoopRamp, pidCoefs, 0);
-    }
```

Comment:
The config is saved in the code changing it is irrelavent for the Json (and usually when you change it you don't want it to stay that way).
why not have the ability to change the settings? becuase you already can change a sepcific setting, pid with setP and such, openLoopRamp with configOpenloopRamp and so on.
want to change multiple stuff? 1. why would you? 2. the way it's written here you have to change all settings because it resets what you don't change to defualt by my understanding.